.travis.yml

```diff
@@ -11,7 +11,10 @@ cache:
 jobs:
   include:
   - stage: lint
-    script: yarn lint
+    script:
+    - ./tools/check-signed-off.sh
```

Comment:
I think you'll need `./tools/check-signed-off.sh --travis` here. When merging a PR, Travis tends to rewrite the author info of the commit with the info found in the public profile of the PR's author. Which can be different from the settings found on the local machine of the author. At least that was the case some time back and caused all validations to check out perfectly while Travis was running for the PR but going red as soon as patches got merged into master. The solution (workaround?) was to be strict when checking the PR but be tolerant when the PR gets merged. (That works if master can progress only through PRs.)